README.md

```diff
@@ -1 +1,65 @@
-# Ruby-Capstone-Project
\ No newline at end of file
+# Ruby-Capstone-Project

```

Comment:
It's great you have added a video link in the PR description. It was nice to see all of you. 

- [ ] [Optional] The video should contain a highlight about a piece of the code the student finds interesting or is proud of making. You talked about the rainbow use but if that's your favorite part, it would be great if you can add how you did it.

Reply:
Thank you for your feedback @arslanbisharat. This is noted.